deploy/helm/wg-access-server/values.yaml

```diff
@@ -18,6 +18,7 @@ wireguard:
   service:
     sessionAffinity: ClientIP
     type: ClusterIP
+    port: 51820
```

Comment:
I don't see `.Values.wireguard.service.port` referenced anywhere, is this really option really used?

Reply:
You're right that was some experimentation from my side, I forgot that I left it in.
I've also wanted to add functionality to change the port. Should I open a separate PR for these changes?

Reply:
If it's a bigger thing please open a new PR, but if it just changes two lines like the tag you might as well attach it to this one.

Reply:
It would add one line in `values.yaml` and change two lines in `service.yaml` and 3 in `deployment.yaml`. So it's fine I guess?